lib/round.js

```diff
@@ -7,8 +9,9 @@ export class RoundData {
    */
   constructor (index) {
     this.index = index
-    /** @type {import('./preprocess.js').Measurement[]} */
+    /** @type {string[]} */
     this.measurementBatches = []
+    /** @type {Measurement[]} */
```

Comment:
Surprising to me that the type checker didn't catch this mistake

Reply:
Before this change, the type was `any` - essentially there was no typechecking for this property.